error.go

```diff
@@ -0,0 +1,5 @@
+package tprstorage
+
+import "github.com/giantswarm/microstorage"
+
+var invalidConfigError = microstorage.InvalidConfigError
```

Comment:
I fixed the mess and added comment why it's like that: https://github.com/giantswarm/tprstorage/pull/1/commits/db51600ad4fdc4cbe281a8d81937ee730b9e7171